content/hi/version_control.md

```diff
@@ -0,0 +1,17 @@
+---
+title: संस्करण नियंत्रण
```

Comment:
```suggestion
title: संस्करण नियंत्रण (Version Control)
```

Reply:
Sorry, missed this in earlier reviews, we always prefer to add English in brackets for the title.